semcore/d3-chart/src/a11y/PlotA11yView.tsx

```diff
@@ -0,0 +1,182 @@
+      .map((entry) => entry!);
+    const doublicated: Record<string, true> = {};
+    const handled: Record<string, string | number> = {};
+    for (const [key, label] of baseKeys) {
+      if (handled[label]) {
+        doublicated[key] = true;
+        doublicated[handled[label]] = true;
+      }
+      handled[label] = key;
+    }
+    return doublicated;
+  }, [keys, hints]);
+  const renderTitle = React.useCallback(
+    (dataKey: string | number) => {
+      if (hints.titles.valuesAxes[dataKey]) return hints.titles.valuesAxes[dataKey];
+      if (!duplicatedBaseKeys[dataKey]) {
+        if (hints.fields.verticalAxes.has(dataKey) && hints.axesTitle.vertical)
+          return hints.axesTitle.vertical;
+        if (hints.fields.horizontalAxes.has(dataKey) && hints.axesTitle.horizontal)
+          return hints.axesTitle.horizontal;
+      }
+
+      return dataKey;
+    },
+    [data, hints, duplicatedBaseKeys],
+  );
+
+  const [summary, setSummary] = React.useState<string | null>(null);
+  const [generatingSummary, setGeneratingSummary] = React.useState(true);
+
+  React.useEffect(() => {
+    if (config.disable) {
+      setSummary('');
+      setGeneratingSummary(false);
+      return;
+    }
+    if (config.override) {
+      setSummary(config.override);
+      setGeneratingSummary(false);
+      return;
+    }
+    setTimeout(() => {
+      setSummary(summarize(providedData, hints, config, locale));
+      setGeneratingSummary(false);
+    }, 0);
+  }, [providedData, hints, config, locale]);
+
+  const handleSkip = React.useCallback(() => {
+    if (!plotRef.current) return;
+
+    heavyFindNextFocusableElement(plotRef.current)?.focus();
+  }, []);
+  const handleSkipKeyboard = React.useCallback(
+    (event: React.KeyboardEvent) => {
+      if (event.code !== 'Enter') return;
+
+      handleSkip();
+    },
+    [handleSkip],
+  );
+  const handleGoToTable = React.useCallback(() => {
+    dataTableRef.current?.focus();
+  }, []);
+  const handleGoToTableKeyboard = React.useCallback(
+    (event: React.KeyboardEvent) => {
+      if (event.code !== 'Enter') return;
+
+      handleGoToTable();
+    },
+    [handleGoToTable],
+  );
+
+  const texts = React.useMemo(
+    () => ({
+      label: intl.formatMessage(
+        { id: 'view-label' },
+        { plotLabel: plotLabel ?? intl.formatMessage({ id: 'view-default-plot-label' }) },
+      ),
+      close: intl.formatMessage({ id: 'view-close' }),
+      skipPlot: intl.formatMessage({ id: 'view-skip-plot' }),
+      goToTable: intl.formatMessage({ id: 'view-go-to-table' }),
+      summary: intl.formatMessage({ id: 'view-data-summary' }),
+      summaryPlaceholder: intl.formatMessage({ id: 'view-summary-placeholder' }),
+      table: intl.formatMessage({ id: 'view-table' }),
+    }),
+    [intl, plotLabel],
+  );
+
+  const dataTableRef = React.useRef<HTMLTableElement>(null);
+
+  return sstyled(styles)(
+    <SPlotA11yView render="div" tabIndex={0} aria-label={texts.label}>
```

Comment:
I understand that this is not what we need, but let's make a render from the Box component because I see a lot of errors in the console.
<img width="1015" alt="Screenshot 2022-08-19 at 14 49 21" src="https://user-images.githubusercontent.com/4582799/185612905-2e06a1ef-d3ae-4f64-ba83-9b345cba0b75.png">